test/e2e/setup.go

```diff
@@ -75,15 +75,25 @@ var (
 	vnetResourceGroup string
 	clusterName       string
 	clients           *clientSet
+
+	dockerSucceeded bool
 )
 
 func skipIfNotInDevelopmentEnv() {
+	// docker cmds will fail in INT until we figure out a solution since
```

Comment:
We can move this comment into the separate function.

Reply:
yes I messed up the refactoring haha, should be fine now